index.js

```diff
@@ -28,13 +28,24 @@ var crypto = require('crypto');
 
 
 // Defaults
+var server = "share1.dexcom.com"
```

Comment:
the other bits of code uses semicolons to terminate the lines, could use it here too

Reply:
fixed